cli/src/ios/update.ts

```diff
@@ -96,8 +96,8 @@ export async function updatePodfile(
   const podfileLockPath = join(projectRoot, 'Podfile.lock');
   let podfileContent = await readFileAsync(podfilePath, 'utf8');
   podfileContent = podfileContent.replace(
-    /(Automatic Capacitor Pod dependencies, do not delete)[\s\S]*(# Do not delete)/,
-    '$1' + dependenciesContent + '\n  $2',
+    /(def capacitor_pods)[\s\S]+?(\nend)/,
+    `$1\n  # Autogenerated by Capacitor--DO NOT DELETE${dependenciesContent}\n  # DO NOT DELETE$2`,
```

Comment:
@jcesarmobile Here's what I ended up doing. Now it replaces the content of the entire block. I think this is okay?